### DISCUSSION TITLE
Problem with inputs when starting a new build

### DISCUSSION THREAD

**Discussion by VladMomotCeladon** [codemagic-ci-cd/codemagic-docs#2834] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

When I try to start a new build not using the gitlab webhook but inside codemagic, I can't do it because an error appears for any boolean input, for example for submitToSlack: Configuration error in workflow "react-native-android-uat":
Environment variable ${{ inputs.submitToSlack }} used in "workflows -> react-native-android-uat -> publishing -> slack -> notify -> success" cannot be used, expected "true" or "false", but got "${{ inputs.submitToSlack }}". This configuration worked recently but now it doesn't. Maybe the code logic for them has been updated or is this a bug on the codemagic side? What to do in this case?

### Expected results

A switch will appear for this field

### Actual results

Configuration error in workflow "react-native-android-uat": 
	Environment variable ${{ inputs.submitToSlack }} used in "workflows -> react-native-android-uat -> publishing -> slack -> notify -> success" cannot be used, expected "true" or "false", but got "${{ inputs.submitToSlack }}"

**Comment by VladMomotCeladon** [comment]
<img width="557" alt="image" src="https://github.com/user-attachments/assets/3719fe8d-2812-4a23-b083-25db11ce95d7">

**Reply by priitlatt** [reply, marked as answer]
@VladMomotCeladon thank you for reporting this. That was an issue on our side that has since been patched. We're terribly sorry for the trouble.

**Reply by VladMomotCeladon** [reply]
Thanks, yes, now it works
